content/Mobile_Dev/WebAuthn/index.adoc

```diff
@@ -0,0 +1,22 @@
+== WebAuthn development for mobile devices
+At this stage you have selected WebAuthn as your primary MFA option for your mobile solution. Through the rest of this series we are going to cover a variety of different steps and concepts that will help you along your implementation journey. Don’t be frightened if you are not familiar with WebAuthn, we are going to begin this section by giving an overview on what WebAuthn is, along with more resources for you to take a deeper dive into the concepts, and a how-to on deploying a sample application. Once you feel ready then proceed to the following steps where we will dive into OS and ecosystem specific concepts. 
+
+=== What if I'm not familiar with WebAuthn?
+If you are not familiar with WebAuth, and are looking to dive into the vast array of concepts, then please use our link:/Mobile_Dev/WebAuthn/WebAuthn_Primer.html[WebAuthn primer for mobile development]. This will provide a variety of resources including: links to helpful documentation, considerations for mobile development, and a guide for deploying your own sample application.
+
+=== Do I need OS specific client applications when my users are using iOS vs Android?
+As with all things in technology the answer is “it depends”. If you develop a web application that leverages WebAuthn, the answer is no. Your users will be able to access the same experience across different mobile operating systems. It’s worth noting that some browsers may not have specific WebAuthn functionality built into it at the moment. Ensure you consult our link:https://developers.yubico.com/WebAuthn/WebAuthn_Browser_Support/[WebAuthn compatibility matrix] to ensure that you understand what features are/aren’t available in specific browser + OS combinations.
+
+If your users are going to be using your application from a native application, then the answer is yes, you will need specific client apps for each mobile OS. Android and iOS each allow for a degree of support for WebAuthn in native applications. The nuances will be covered in the OS specific guides in the following section. 
+
+=== OS specific guides for WebAuthn
```

Comment:
The developer site has some existing iOS WebAuthn guidance. Should we link to that here?

Reply:
https://developers.yubico.com/WebAuthn/Supporting_U2F_or_FIDO2_Security_Keys_on_iOS_or_iPadOS/

Reply:
https://developers.yubico.com/Mobile/

Reply:
I opt'd to do two things here:
* Added the first link to the list of guides
* Instead of pointing to the full Mobile directory, I linked to the Android specific section

I didn't want the web guidance in /Mobile to conflict or confuse with the guidance in this series

Let me know what you think